sklearn/naive_bayes.py

```diff
@@ -137,16 +147,40 @@ class GaussianNB(BaseNB):
     >>> from sklearn.naive_bayes import GaussianNB
     >>> clf = GaussianNB()
     >>> clf.fit(X, Y)
-    GaussianNB()
+    GaussianNB(class_prior=None, fit_prior=True)
     >>> print(clf.predict([[-0.8, -1]]))
     [1]
     >>> clf_pf = GaussianNB()
     >>> clf_pf.partial_fit(X, Y, np.unique(Y))
-    GaussianNB()
+    GaussianNB(class_prior=None, fit_prior=True)
     >>> print(clf_pf.predict([[-0.8, -1]]))
     [1]
     """
 
+    def __init__(self, fit_prior=True, class_prior=None):
+        self.fit_prior = fit_prior
+        self.class_prior = class_prior
+
+    def _update_class_prior(self, class_prior=None):
```

Comment:
It might be a good idea to rewrite `BaseDiscreteNB` as well at some point of time :-)